buildenv/jenkins/perfPipeline.groovy

```diff
@@ -99,16 +102,29 @@ timestamps {
                                 // test
                                 testParams << string(name: "TEST_NUM", value: "TEST_NUM" + i.toString())
                                 def testRun = triggerJob("${item.BENCHMARK}", "${platform}", testParams, "test")
-                                aggregateLogs(testRun)
+                                aggregateLogs(testRun, testRuntimes)
 
                                 // baseline
                                 if (RUN_BASELINE) {
                                     baselineParams << string(name: "BASELINE_NUM", value: "BASELINE_NUM_" + i.toString())
                                     def baseRun = triggerJob("${item.BENCHMARK}", "${platform}", baselineParams, "baseline")
-                                    aggregateLogs(baseRun)
+                                    aggregateLogs(baseRun, baselineRuntimes)
                                 } else {
                                     echo "Skipping baseline run since RUN_BASELINE is set to false"
                                 }
+
+                                testStats = stats(testRuntimes)
+                                baselineStats = stats(baselineRuntimes)
+                                def score = (testStats.mean/baselineStats.mean) * 100
```

Comment:
json can contain multiple benchmarks. We cannot assume we only have one (i.e., `dacapo-h2`). I am ok to deliver this as the initial version and add the enhancement in the future PR.